src/extend-utils.ts

```diff
@@ -75,3 +77,13 @@ export function bigIntToStringUtil(convertableBigInt: BigInt) {
 }
 
 
+export function getWalletUtil(name: string, hre: HardhatRuntimeEnvironment) {
+    const wallet = hre.config.wallets[name];
+    if (!wallet) {
+        const available = Object.keys(hre.config.wallets).join(", ");
+        const msg = `Invalid wallet provided: ${name}.\nValid wallets: ${available}`;
```

Comment:
The wallet name is what's invalid, not the wallet itself.

Reply:
Here it means that if the wallet name is not in the hardhat.config file, the wallet with that name doesent exist, so "wallet" will be undefined